interface/src/main/java/xsbti/api/Modifiers.java

```diff
@@ -10,7 +10,7 @@
 	private static final int LazyBit = 5;
 	private static final int MacroBit = 6;
 
-	private static final int flag(boolean set, int bit)
```

Comment:
I'd lean on the side of not touching anything in xsbti right now.   This code is used in unexpected ways (intellij may not be aware) and it DOES do dirty things.....